pages/community-lessons.md

```diff
@@ -22,69 +22,94 @@ permalink: "/community-lessons/"
 
 The HPC Carpentry community is committed to a collaborative and open process for
 lesson development and to sharing teaching materials. We
-provide two avenues for community members to share lesson
-materials - **The Carpentries Incubator** and **HPC Carpentry Lab**.
+provide three avenues for community members to share lesson
+materials - **The Carpentries Incubator**, **HPC Carpentry** and our
+**HPC Carpentry Listings**.
 
 [The Carpentries Incubator](#the-carpentries-incubator) is for:
 * Collaborative lesson development (from conceptual to stable lessons).
-* Providing visibility for lessons that are being worked on.
+* Providing wider visibility for lessons that are being worked on.
 
-[HPC Carpentry Lab](#hpc-carpentry-lab) is for:
+[HPC Carpentry](#hpc-carpentry) is for:
 * Repository of peer-reviewed, short-format, lessons that use the teaching
   approach and lesson design from The Carpentries.
 * Getting peer-review on the content of the lesson in the way traditional
   journal peer-review wouldn’t be able to provide.
 
+[HPC Carpentry Listings](#hpc-carpentry-listings) is for:
```

Comment:
I like it! ... though, if these are beyond our organization, I suggest "HPC Listings" or "HPC Resources" instead.